src/compile.coffee

```diff
@@ -73,7 +73,7 @@ findRelevantTransactions = (mediaType, parseResult) ->
         relevantTransactions.push(httpTransaction) unless httpTransaction.attributes.example is example
       else
         relevantTransactions.push(httpTransaction)
-      example = httpTransaction.attributes.example
+      example = httpTransaction.attributes?.example
```

Comment:
Attributes won't be included in elements if it is empty anymore. I did see some checks for attributes but some we're missing that I have added.